homeassistant/components/overkiz/binary_sensor.py

```diff
@@ -109,17 +109,20 @@ class OverkizBinarySensorDescription(BinarySensorEntityDescription):
         key=OverkizState.CORE_HEATING_STATUS,
         name="Heating status",
         device_class=BinarySensorDeviceClass.HEAT,
-        value_fn=lambda state: state == OverkizCommandParam.ON,
+        value_fn=lambda state: cast(str, state).lower()
```

Comment:
Why do you need the `.lower()` here? Does your device leverage 'Heating' as a string instead of 'heating'?

Reply:
Yes, the bugfix is in making the state lowercase, as `Heating` gets compared with `heating`.